e2e/node_test.go

```diff
@@ -37,7 +37,12 @@ func testNode() {
 				return err
 			}
 
-			if len(podlist.Items) != 3 {
+			count := 1
+			if !isLvmdEnv() {
+				count = 3
+			}
+
+			if len(podlist.Items) != count {
```

Comment:
I assume this test checks topolvm-node should be deployed on all nodes,
so check if pod number equals worker node number.